packages/dialect-react/index.ts

```diff
@@ -1,10 +1,38 @@
-export * from './components/ApiContext';
-export * from './components/DialectContext';
-export * from './api';
-export * as DialectErrors from './utils/errors';
-export { connected } from './utils/helpers';
-
+import * as DialectErrors from './utils/errors';
```

Comment:
seems like one wildcard is still here

Reply:
or this kind of named wildcard is not a problem?

Reply:
This one is fine, since it's an `import` wildcard first and a named export then (at least I hope that it's fine)

Reply:
I think the problem is more related to an export wildcard